packages/e2e/detox/init.ts

```diff
@@ -23,8 +23,6 @@ BeforeAll(async () => {
   await init({
     testRunnerArgv: { ...process.env },
   });
-  // android only, no-op for ios, required for mock server connection
-  await device.reverseTcpPort(9091);
```

Comment:
Moved to CI because this is run before every feature, not before all tests as I originally thought.